src/ophyd_async/core/device_save_loader.py

```diff
@@ -1,35 +1,30 @@
 from enum import Enum
 from functools import partial
-from typing import Any, Callable, Dict, Generator, List, Optional, Sequence, Union
+from typing import Any, Callable, Dict, Generator, List, Optional, Sequence
 
 import numpy as np
 import numpy.typing as npt
 import yaml
 from bluesky.plan_stubs import abs_set, wait
 from bluesky.protocols import Location
 from bluesky.utils import Msg
-from epicscorelibs.ca.dbr import ca_array, ca_float, ca_int, ca_str
+from epicscorelibs.ca.dbr import ca_array
 
 from .device import Device
 from .signal import SignalRW
 
-CaType = Union[ca_float, ca_int, ca_str, ca_array]
-
 
 def ndarray_representer(dumper: yaml.Dumper, array: npt.NDArray[Any]) -> yaml.Node:
     return dumper.represent_sequence(
         "tag:yaml.org,2002:seq", array.tolist(), flow_style=True
     )
 
 
-def ca_dbr_representer(dumper: yaml.Dumper, value: CaType) -> yaml.Node:
+def ca_dbr_representer(dumper: yaml.Dumper, value: ca_array) -> yaml.Node:
```

Comment:
Should: Do we need this function now? Can't we just remove it for the change suggested below...